orbitize/nbody.py

```diff
@@ -16,104 +28,115 @@ def calc_orbit(
         inc (np.array): inclination [radians]
         aop (np.array): argument of periastron [radians]
         pan (np.array): longitude of the ascending node [radians]
-        tau (np.array): epoch of periastron passage in fraction of orbital period 
+        tau (np.array): epoch of periastron passage in fraction of orbital period
             past MJD=0 [0,1]
         plx (np.array): parallax [mas]
-        mtot (np.array): total mass of the two-body orbit (M_* + M_planet) 
+        mtot (np.array): total mass of the two-body orbit (M_* + M_planet)
```

Comment:
should `mtot` be an float, instead of an array?

Reply:
yes, good catch. Thanks!